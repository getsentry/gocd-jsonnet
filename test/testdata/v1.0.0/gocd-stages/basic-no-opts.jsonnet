local gocd_stages = import '../../../../v1.0.0/gocd-stages.libsonnet';

gocd_stages.basic('example', [])
