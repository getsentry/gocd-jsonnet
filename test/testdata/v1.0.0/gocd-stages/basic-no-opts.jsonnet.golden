{
   "example": {
      "fetch_materials": true,
      "jobs": {
         "example": {
            "tasks": [ ]
         }
      }
   }
}
